integration_tests/plugins/test_ruby_plugin.py

```diff
@@ -33,6 +33,14 @@ def test_ruby_gem_install_rack(self):
 
     def test_ruby_hello(self):
         self.run_snapcraft('stage', 'ruby-hello')
+        os.environ['RUBYPATH'] = os.path.join(self.stage_dir, 'bin')
```

Comment:
Why did you change it this way? Maybe this needs more discussion. How about opening a forum topic?

The problem I'm seeing with the approach is that it's not testing the real snap anymore. Integration tests should work in a way that you can build, install and run the same test snap - but the variables are set in the test itself, so there's no guarantee it works outside the test.